packages/main/src/components/Toolbar/Toolbar.jss.ts

```diff
@@ -85,23 +85,26 @@ export const styles = {
   popoverPhone: {
     width: 'calc(100% - 10px)',
     maxWidth: 'calc(100% - 10px)',
-    left: '5px !important'
+    insetInlineStart: '5px !important'
   },
   popoverContent: {
     padding: CssSizeVariables.ui5WcrToolbarPopoverContentPadding,
     display: 'flex',
     flexDirection: 'column',
-    '& [ui5-toggle-button], & [ui5-button]': {
-      marginBottom: '0.25rem'
+    '& >[ui5-toggle-button], & >[ui5-button]': {
+      marginBlock: '0.25rem'
```

Comment:
Are we now using a two sided margin? or should this rather be `marginBlockEnd`?

Reply:
Yup. Checked with SapUI5 and they do the same